src/Navigation.js

```diff
@@ -1,131 +1,52 @@
-                            </Card>
-                        </Accordion>
-                    </Nav.Item>
-                    <Nav.Item>
-                        <Accordion>
-                            <Card>
-                                <Card.Header>
-                                    <Accordion.Toggle as={Link} to="/portfolio" eventKey="0">My Space</Accordion.Toggle>
-                                </Card.Header>
-                                <Accordion.Collapse eventKey="0">
-                                    <Nav>
-                                        <Nav.Link as={Link} to="/portfolio">Portfolio</Nav.Link>
-                                        <Dropdown>
-                                            <Dropdown.Toggle variant="light">Profile</Dropdown.Toggle>
-                                            <DropdownMenu>
-                                                <Nav.Link as={Link} to="/personal-details" eventKey="0">Personal Details</Nav.Link>
-                                                <Nav.Link as={Link} to="/additional-info" eventKey="0">Additional Info</Nav.Link>
-                                                <Nav.Link as={Link} to="/personal-background" eventKey="0">Personal Background</Nav.Link>
-                                            </DropdownMenu>
-                                        </Dropdown>
-                                        <Nav.Link as={Link} to="/request-history">Request History</Nav.Link>
-                                        <Nav.Link as={Link} to="/organization-profile" eventKey="0">My Organization</Nav.Link>
-                                    </Nav>
-                                </Accordion.Collapse>
-                            </Card>
-                        </Accordion>
-                    </Nav.Item>
-                    {!isAuth ?
-                        <>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/register" eventKey="0">Register</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/login" eventKey="0" onClick={login}>Login</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                        </>
-                        :
-                        <Nav.Item>
-                            <Card>
-                                <Card.Header>
-                                    <Accordion.Toggle as={Link} to="/" eventKey="0" onClick={logout}>Logout</Accordion.Toggle>
```

Comment:
Make sure this `Logout` option is shown for users who have successfully logged-in (instead of Sign In and Sign Up buttons).